docs/src/main/sphinx/release/release-466.md

```diff
@@ -0,0 +1,112 @@
+# Release 466 (27 Nov 2024)
+
+## General
+
+* Improve performance of queries with grouping on joins. ({issue}`23812`)
+* TBD Spooling protocol is stable. Link to docs! ({issue}`24214`)
+* TBD Improve performance of querying `system.jdbc.tables` for multiple catalogs. ({issue}`24159`)
+* Support partial update. ({issue}`24075`)
```

Comment:
@electrum @djsstarburst @chenjian2664 .. do we somehow reword this to be more clear what it actually adds? Or leave it out and just announce the merge support in the connectors when the related PR is merged?

Reply:
this is not user-visible change, it allows better planning of MERGE queries in JDBC connectors as far as I understand

Please confirm with @djsstarburst and @electrum though.

Reply:
Clarified and updated already.